src/graphql/graphql.ts

```diff
@@ -1118,3 +1118,375 @@ export function createIncrementalReadersV2(
   }
   return result;
 }
+
+/**
+ * Converts a V1 query into incremental:
+ * Adds "from" and "to" query variables.
+ * Adds a filter "from" <= refreshedAt < "to" to the top level model.
+ * Makes sure metadata { refreshedAt } is selected.
+ *
+ * Example:
+ *  vcs {
+ *     pullRequests {
+ *       nodes {
+ *         title
+ *       }
+ *     }
+ *  }
+ *
+ * becomes:
+ *  query incrementalQuery($from: builtin_BigInt!, $to: builtin_BigInt!) {
+ *   vcs {
+ *     pullRequests(
+ *       filter: {
+ *        refreshedAtMillis: {
+ *          greaterThanOrEqualTo: $from,
+ *          lessThan: $to
+ *        }
+ *       }
+ *     ) {
+ *       nodes {
+ *         title
+ *         metadata {
+ *           refreshedAt
+ *         }
+ *       }
+ *     }
+ *   }
+ *  }
+ */
+export function toIncrementalV1(query: string): string {
```

Comment:
What replaces `nodes` with `edges`? e.g what converts the query into a paginated one (same question for v2)

Reply:
Great!